galera/src/replicator_str.cpp

```diff
@@ -338,49 +338,6 @@ ReplicatorSMM::donate_sst(void* const         recv_ctx,
     wsrep_cb_status const err(sst_donate_cb_(app_ctx_, recv_ctx,
                                              streq.sst_req(), streq.sst_len(),
                                              &state_id, 0, 0, bypass));
-<<<<<<< HEAD
-
-    /* The fix to codership/galera#284 may break backward comatibility due to
```

Comment:
I don't see codership fixing this differently in the commit history. If I missed it, please mention it in the commit message. And if I didn't, and it really isn't there, this shouldn't be removed.

Reply:
Marked as resolved since we decided to go revert our fix and go with upstream's fix.